golden_response/staging/get_landing_page_data/tha.json

```diff
@@ -2225,45 +2225,7 @@
         },
         "provenanceDomain": "worldbank.org"
       },
-      "GiniIndex_EcconomicActivity": {
-        "data": {
-          "1981": 58,
-          "1982": 58.4,
-          "1983": 59,
-          "1984": 58.4,
-          "1985": 55.6,
-          "1986": 58.5,
-          "1987": 59.7,
-          "1988": 61.4,
-          "1989": 63.3,
-          "1990": 60.5,
-          "1992": 53.2,
-          "1993": 60.1,
-          "1995": 59.6,
-          "1996": 59.9,
-          "1997": 59.8,
-          "1998": 59.6,
-          "1999": 59,
-          "2001": 58.4,
-          "2002": 58.1,
-          "2003": 57.6,
-          "2004": 56.5,
-          "2005": 56.3,
-          "2006": 55.6,
-          "2007": 54.9,
-          "2008": 54,
-          "2009": 53.7,
-          "2011": 52.9,
-          "2012": 53.5,
-          "2013": 52.8,
-          "2014": 52.1,
-          "2015": 51.9,
-          "2016": 53.3,
-          "2017": 53.3,
-          "2018": 53.9
-        },
-        "provenanceDomain": "worldbank.org"
-      },
+      "GiniIndex_EcconomicActivity": {},
```

Comment:
This is a typo that needs to be fixed in https://github.com/datacommonsorg/website/blob/master/server/chart_config.json#L514

I'd fixed in google3 when I built cache.